packages/qwik/src/server/utils.ts

```diff
@@ -27,7 +27,7 @@ export function getBuildBase(opts: RenderToStringOptions) {
     }
     return base;
   }
-  return '/build/';
+  return `${import.meta.env.BASE_URL}build/`;
```

Comment:
```suggestion
  return `${import.meta.env.BASE_URL || '/'}build/`;
```

Reply:
Isn't that always defined?

Reply:
nope

Reply:
So this [is determined by](https://vitejs.dev/guide/env-and-mode#:~:text=This%20is%20determined%20by%20the%20base%20config%20option.) [the base config option](https://vitejs.dev/config/shared-options.html#base) which defaults to `/`, and we check that it ends on a `/`, so I think it's always defined.

Reply:
can we still do warning for now and throw errors later